multihost_test/Sanity/test_pam_pwquality.py

```diff
@@ -0,0 +1,146 @@
+
+import pytest
+import subprocess
+import os
+
+
+def exceute_cmd(multihost, command):
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+@pytest.mark.tier1_2
+class TestPamPwquality(object):
+    def test_pam_pwquality1(self, multihost, create_localusers):
```

Comment:
It would be nice to rename the function name with a more meaningful name, or to include some comment describing what the test is doing

Reply:
done